### PR TITLE
csv: csv-processor: don't hardcode annotations

### PR DESCRIPTION
**Pull request by fromanirh**
We have support to inject custom annotations from
a JSON file. So let's use this facility in our default
flow instead of hardcoding them in the tool.

This also fixes a typo found in the hardcoded annotation.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
we will likely need a backport to 4.6

File: hack/csv-generate.sh [hunk 1]
```diff
@@ -83,8 +83,8 @@ if [[ ${CSV_VERSION} =~ 4.7.* ]]; then
     --replaces-csv-version "${REPLACES_CSV_VERSION}" \
     --skip-range "${CSV_SKIP_RANGE}" \
     --min-kube-version "${MIN_KUBE_VERSION}" \
-    "${MAINTAINERS}" \
-    "${EXTRA_ANNOTATIONS}"
+    "${EXTRA_ANNOTATIONS}" \
```
Comment (marcel-apf) [inline hack/csv-generate.sh:86]: You changed the order, in purpose?

Comment (fromanirh) [reply hack/csv-generate.sh:86]: yes, and also renamed. The order is not really relevant here. But I can split to a different commit if needed.

Comment (marcel-apf) [reply hack/csv-generate.sh:86]: No need, thanks

**Comment by fromanirh** [comment]
/hold
I need Yanir's lgtm for awareness

**Pull request merged**